cmd/service-ca-operator-tests-ext/main.go

```diff
@@ -9,16 +9,21 @@ package main
 
 import (
 	"context"
+	"fmt"
```

Comment:
for easier review:

please open a new PR just for the changes to this file.

Reply:
once you open that PR please also remove https://github.com/openshift/service-ca-operator/tree/main/test/.openshift-tests-extension